environment-dev.yml

```diff
@@ -19,3 +19,16 @@ dependencies:
   - pyyaml
   - cli11
   - pytest
+    #  - krb5-static
+    #  - libcurl-static 7.76.1 *_0
+    #  - libnghttp2-static
+    #  - libopenssl-static
+    #  - libxml2
+    #  - libsolv-static >=0.7.18
+    #  - libssh2-static
+    #  - lz4-c-static
+    #  - reproc-cpp-static
+    #  - reproc-static
+    #  - yaml-cpp-static
+    #  - xz-static
+    #  - zstd-static
```

Comment:
These were all the packages required to build a static micromamba. There were also a few other tweaks I had to do to get it to build. What is your process of building a static micromamba?

Reply:
Also, these could probably be removed or moved to a different environment file.

Reply:
Hey! Have a look at recently added https://github.com/mamba-org/mamba/blob/master/environment-static-dev.yml